File: C-Fundamentals/03 Operators and Expressions/Point in a circle/Point in a circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Point_in_a_circle
{
   public class Program
    {
       public static void Main(string[] args)
        {
            // Вход - приема две дробни числа, като координати на точка.
            double x = double.Parse(Console.ReadLine());
            double y = double.Parse(Console.ReadLine());

            // Обработка, изход - изчислява и показва, дали точката с входните координати се намира в окръжността с
            // център 0,0 и радиус 2 .
            double z = Math.Sqrt(x*x + y*y);
            if (z > 2)
            {
                Console.WriteLine("no {0:f2}" , z);
            }
            else Console.WriteLine("yes {0:f2}", z);

        }
    }
}

File: JS fundamentals Homework/04. Operators and Expressions/Point in a Circle.js
function solve(args) {
    var x  = + args[0],
        y = + args[1],
        radius = Math.sqrt(x*x + y*y).toFixed(2);

    if(radius <= 2){
        return ( "yes " + radius );
    }
    else {
        return ( "no " + radius );
    }
}
